data/test/scenarios/wml_tests/UnitsWML/AbilitiesWML/teleport/teleport_simple.cfg

```diff
@@ -0,0 +1,59 @@
+# wmllint: no translatables
+
+#####
+# API(s) being tested: [teleport]
+##
+# Actions:
+# Give side 1 units the ability to teleport that applies to themselves between owned villages
```

Comment:
They aren't owned in this test. The test passes because the movement captures 4,2 but doesn't capture 1,2.

Reply:
Updated.